lib/main_common.pm

```diff
@@ -1525,6 +1510,7 @@ sub load_extra_tests_console {
     loadtest "console/salt" if (is_jeos || is_opensuse);
     loadtest "console/gpg";
     loadtest "console/rsync";
+    loadtest "console/clamav";
```

Comment:
This change is definitely not intended.

Reply:
I guess someone pushed on my branch - repushed my changes and disabled push access from maintainers to my branch.